src/index.html

```diff
@@ -1,13 +1,457 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>MyBiKE</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="./images/favicon.svg"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="header__content">
+        <div class="header__nav">
+          <a href="#">
+            <img
+              class="logo__image"
+              src="./images/MyBiKE--black.svg"
+              alt="logo"
+            >
+          </a>
```

Comment:
your logo is not aligned in center
![image](https://user-images.githubusercontent.com/37161315/197524272-cbb70544-c8b6-43dc-ad7d-45864d0061f6.png)